public/javascripts/add.js

```diff
@@ -0,0 +1,10 @@
+//to show one stanza box on the form
+song.lyric.push([''])
+var obj = {
+    song: song,
+    url: '/songlist-db/add',
+    readonly: false,
+    submitButton: true,
+    formID: ''
+}
```

Comment:
this just feels weird to me...why is the formId blank? And why is this called javascripts/add.js ... it does not make much sense why the file is named add.js